site/content/src/main/exercises/stdlib/Constructors.scala

```diff
@@ -0,0 +1,115 @@
+package stdlib
+
+import org.scalatest._
+
+/** Constructors
+  *
+  * Constructors Description
+  */
+
+object Constructors extends FlatSpec with Matchers with exercise.Section {
+
+
+  /** primaryConstructorConstructors
+    *
+    * Primary Constructor
+    *
+    * In Java we have a no-args default constructor which is provided for every class which doesn't provide its own constructor methods. On a similar lines Primary Constructor in Scala is the kind-of default constructor in the way every class in Scala would have a Primary Constructor.
+    *
+    * The primary constructor spans the complete class definition i.e in the example below the age field gets initialized as part of the Primary Constructor of the Employee class.
+    *
+    *
+    * class Employee {
+    * var age:Int = 20
+    * }
+    *
+    *
+    * In the above example the primary constructor didn't accept any parameters. But the primary constructor can also accepts parameters, this is where it is different from default constructors in Java. Let me not try to draw an analogy any further.
+    *
+    *
+    * class Employee(val firstName:String,
+    * val lastName:String) {
+    *
+    * override def toString():String = {
+    * "First Name: " + firstName + " Last Name: " + lastName
+    * }
+    * }
+    *
+    *
+    * #### Auxiliary Constructor
```

Comment:
Scaladoc uses `=` symbols on both sides for marking headers, we may want to use that syntax.